react-native-purchases-ui/package.json

```diff
@@ -75,7 +75,8 @@
   },
   "peerDependencies": {
     "react": "*",
-    "react-native": "*"
+    "react-native": "*",
+    "react-native-purchases": "8.11.8"
```

Comment:
Keeping the version to the exact version this version of `react-native-purchases` should be compatible with. That way we minimize the chance of someone having a mismatch between both libraries (it will give a warning when installing).